doc/dev/quick-start-devel.rst

```diff
@@ -4,8 +4,36 @@
 Quick start guide for developers
 ################################
 
-Using Qserv with your own custom code or arbitrary versions can be done by
-connecting your local git repository with an eups software stack containing Qserv
-dependencies.
-
+Using Qserv with your own custom code or arbitrary versions can be done using Qserv build containers.
+Full documentation is available here:
 https://confluence.lsstcorp.org/display/DM/Qserv-Lite+Builds+in+Containers
+
+************************************
+Bootstrap a development environement
+************************************
+
+.. code:: sh
+
+    # Clone qserv and its submodules
+    git clone --recursive https://github.com/lsst/qserv
+    cd qserv
+    # Build Qserv base images
+    qserv build-images
+    # Build Qserv use image
+    qserv build-user-build-image
+    # Open a shell in a development container
+    # NOTE: Add '--user=qserv' option to command below if user id equals 1000
+    qserv run-build
+
+
+*****************************
+Build an image for production
+*****************************
+
+.. code:: sh
+
+    # NOTE: Remove build/ directory to restart the build from scratch.
+    # This can fix build errors.
+    rm -rf build/
+    # NOTE: Add '--user=qserv' option to command below if user id equals 1000
+    qserv build -j8
```

Comment:
I proposed squashing the current commit (where the above-shown changes were made) "Install arrow" in order to have a single commit instead of two separate commits:

"Install arrow"
"Document how to set up a development container"

I see no point to have two separate commits each modifying the same Dockerfile and documenting how to use products (containers) built from that extended Dockerfile.

Reply:
The git history is modified according to the remarks you made